docs/source/models/concept_models/vivarium_nutrition_optimization/kids/concept_model.rst

```diff
@@ -1408,10 +1404,46 @@ Wave III
     - Spec. mods
     - Note
   * - 14.0
+    - Update to GBD 2021 data 
+    - Baseline
+    - Baseline
+    - 
+    - 
+  * - 15.0
+    - Change child data to subnational
+    - Baseline
+    - Baseline
+    - 
+    - 
+  * - 16.0
+    - Add targeted SQ-LNS intervention for wasting prevalence > 10%
+    - Baseline
+    - 8
+    - 
+    - Targeted SQ-LNS only
+  * - 17.0
+    - Add other targeted SQ-LNS criteria
```

Comment:
Does this mean additional geography-based targeting algorithms we might be interested in?